code/datums/diseases/critical.dm

```diff
@@ -92,6 +92,12 @@
 	bypasses_immunity = TRUE
 	virus_heal_resistant = TRUE
 
+/datum/disease/has_cure()
+	if (affected_mob.has_status_effect(STATUS_EFFECT_EXERCISED))
```

Comment:
@Fox-McCloud Is this the correct way of approaching this? Excercising is supposed to prevent heart failure for a little while.

Reply:
I just noticed how I actually forgot the path here, whelp.

Reply:
It should be fine just depends on what calls excercise, cuase i know we don't have alot of gym stuff. Also space between if and (). Still going through code.